### PR TITLE
alias: add Russian translation

### PR DESCRIPTION
**Pull request by dremdem**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ru/common/alias.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# alias
+
+> Создает псевдоним -- слова, которые будут заменены командой.
```
Comment (EmilySeville7cfg) [inline pages.ru/common/alias.md:3]:
```suggestion
> Создает псевдонимы -- слова, которые заменяются командой.
```

File: pages.ru/common/alias.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+> Срок действия псевдонима истекает с текущей сессией командной строки если только не определить его в конфигурационном файле, например: `~/.bashrc`.
```
Comment (EmilySeville7cfg) [inline pages.ru/common/alias.md:4]:
```suggestion
> Срок действия псевдонима истекает с окончанием текущей сессии командной строки, если только не определить его в конфигурационном файле, например: `~/.bashrc`.
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages.ru/common/alias.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`alias {{псевдоним}}`
+
+- Удалить псевдоним:
+
+`unalias {{псевдоним}}`
+
+- Превратить `rm` в интерактивную команду:
+
+`alias {{rm}}="{{rm -i}}"`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/alias.md:25]:
```suggestion
`alias {{rm}}="{{rm --interactive}}"`
```

File: pages.ru/common/alias.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Превратить `la` в ссылку на `ls -a`:
```
Comment (EmilySeville7cfg) [inline pages.ru/common/alias.md:27]:
```suggestion
- Превратить `la` в ссылку на `ls --all`:
```

File: pages.ru/common/alias.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`alias {{la}}="{{ls -a}}"`
```
Comment (EmilySeville7cfg) [inline pages.ru/common/alias.md:29]:
```suggestion
`alias {{la}}="{{ls --all}}"`
```

**Review by EmilySeville7cfg — COMMENTED** [review]
Please prefer long options like [here](https://github.com/tldr-pages/tldr/pull/7675). ;)

File: pages.ru/common/alias.md (hunk 3 above)
Comment (navarroaxel) [reply pages.ru/common/alias.md:25]: In this scenario, keep the same example as the main English page.

File: pages.ru/common/alias.md (hunk 5 above)
Comment (navarroaxel) [reply pages.ru/common/alias.md:29]: In this scenario, keep the same example as the main English page.

File: pages.ru/common/alias.md (hunk 4 above)
Comment (navarroaxel) [reply pages.ru/common/alias.md:27]: In this scenario, keep the same example as the main English page.

**Review by navarroaxel — APPROVED** [review]

File: pages.ru/common/alias.md (hunk 1 above)
Comment (dremdem) [reply pages.ru/common/alias.md]: Changed.

File: pages.ru/common/alias.md (hunk 2 above)
Comment (dremdem) [reply pages.ru/common/alias.md]: Changed.

**Pull request merged**